testutil/src/main/java/org/hyperledger/besu/kvstore/AbstractKeyValueStorageTest.java

```diff
@@ -547,4 +582,17 @@ protected byte[] bytesFromHexString(final String hex) {
   protected byte[] bytesOf(final int... bytes) {
     return Bytes.of(bytes).toArrayUnsafe();
   }
+
+  /**
+   * Use random to get a sub folder from the given path, that will not conflict with other folders.
+   * This is needed because we're using Path via Junit 5 @TempDir annotation, and Path doesn't have
+   * a way to create a new tmp folder.
+   *
+   * @param folder the folder in which to create the sub folder
+   * @return the path representing the sub folder
+   */
+  @NotNull
+  protected Path getTempSubFolder(final Path folder) {
+    return folder.resolve(Bytes.random(9).toString());
```

Comment:
Cannot [java.nio.file.Files#createTempDirectory](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createTempDirectory-java.nio.file.Path-java.lang.String-java.nio.file.attribute.FileAttribute...-) be used in this case?

Reply:
Yes! thanks @diega - changed it over